code/modules/uplink/uplink_items.dm

```diff
@@ -1815,14 +1837,29 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/box/hug/reverse_revolver
 	restricted_roles = list("Clown")
 
+/datum/uplink_item/role_restricted/clownpin
+	name = "Ultra Hilarious Firing Pin"
+	desc = "A firing pin that, when inserted into a gun, makes that gun only usable by clowns and clumsy people, and makes that gun honk whenever anyone tries to fire it."
```

Comment:
wait, I don't think that extra comma you added was needed, there are only two "items" in that list